checks.d/jenkins.py

```diff
@@ -75,13 +105,19 @@ def _get_build_metadata(self, dir_name):
 
     def _get_build_results(self, instance_key, job_dir):
         job_name = os.path.basename(job_dir)
-
         try:
             dirs = glob(os.path.join(job_dir, 'builds', '*_*'))
+            # versions of Jenkins > 1.597 dropped the timestamp-named folders
+            if len(dirs) == 0:
+                dirs = glob(os.path.join(job_dir, 'builds', '[0-9]*'))
             if len(dirs) > 0:
-                dirs = sorted(dirs, reverse=True)
+                # versions of Jenkins > 1.597 need to be sorted by build number (integer)
```

Comment:
Could you add in the comments the old and new folder names ? It would be easier to understand your code.